integration-tests/deployment/keystone/types.go

```diff
@@ -130,6 +161,10 @@ func newOcr2Node(id string, ccfgs map[chaintype.ChainType]*v1.ChainConfig, csaPu
 	}
 
 	var sigb [32]byte
+	// initialize sigb to 0s
```

Comment:
Doesn't Go do that already?

Reply:
yes. this is leftover, when i was confused about the prefix we where seeing on chain, which is from the multichain encoding